package.json

```diff
@@ -693,7 +693,7 @@
     "request-promise-native": "^1.0.5",
     "semver": "^5.5.1",
     "vscode-azureextensionui": "^0.17.0",
-    "vscode-extension-telemetry": "0.0.18",
+    "vscode-extension-telemetry": "^0.0.22",
```

Comment:
Just want to make sure you know that `^0.0.22` is functionally the same as `0.0.22` and thus we still won't get any updates automatically. The [caret](https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004) "Allows changes that do not modify the left-most non-zero digit"

I already filed an issue to ask that they change their versioning strategy: https://github.com/Microsoft/vscode-extension-telemetry/issues/22